e2e/cross-teams-report.spec.ts

```diff
@@ -0,0 +1,365 @@
+import { test, expect } from "@playwright/test";
+import { nanoid } from "nanoid";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import "dayjs/locale/fr";
+import { changeReactSelectValue, clickOnEmptyReactSelect } from "./utils";
+
+dayjs.extend(utc);
+dayjs.locale("fr");
+
+test("Cross teams report", async ({ page }) => {
+  // Always use a new items
+  const today = dayjs();
+  const team1Name = nanoid();
+  const team2Name = nanoid();
+  const person1Name = nanoid();
+  const person2Name = nanoid();
+  const person1action = nanoid();
+  const person2action = nanoid();
+  const team1Description = nanoid();
+  const team2Description = nanoid();
+  const team1Collab = nanoid();
+  const team2Collab = nanoid();
+  // const team1Name = "team1Name";
+  // const team2Name = "team2Name";
+  // const person1Name = "person1Name";
+  // const person2Name = "person2Name";
+  // const person1action = "person1action";
+  // const person2action = "person2action";
+  // const team1Description = "team1Description";
+  // const team2Description = "team2Description";
+  // const team1Collab = "team1Collab";
+  // const team2Collab = "team2Collab";
+
+  test.setTimeout(120000);
+
+  await test.step("Log in", async () => {
+    await page.goto("http://localhost:8090/");
+
+    await page.goto("http://localhost:8090/auth");
+
+    await page.getByLabel("Email").click();
+
+    await page.getByLabel("Email").fill("admin1@example.org");
```

Comment:
L'idée c'était d'essayer d'utiliser un admin différent à chaque fois pour être sûr que les tests ne se marchent pas dessus. C'est pas _nécessaire_ mais conseillé.

```suggestion
    await page.getByLabel("Email").fill("admin2@example.org");
```


>  Pour chaque test, on peut utiliser un des 50 admins (idéalement utiliser un admin différent pour chaque test pour éviter les conflits en cas de lancement simultanés de tests).